docs/configuration/monitoring.md

```diff
@@ -1,77 +1,72 @@
 # Monitoring
 
-Currently two monitoring solutions are supported:
+Currently, we support two monitoring solutions:
 
-* [**Crow**](#setting-up-monitoring-using-crow) - A simple, lightweight, and built-in monitoring solution.
-* [**Prometheus + Grafana + Loki**](#setting-up-monitoring-using-grafana-prometheus-and-loki) - A comprehensive third-party monitoring solution, recommended for more advanced usage.
+-   **[Crow](#setting-up-monitoring-using-crow):** A simple, lightweight, and built-in monitoring solution.
+-   **[Prometheus + Grafana + Loki](#setting-up-monitoring-using-grafana-prometheus-and-loki):** A comprehensive third-party monitoring solution, recommended for more advanced usage.
+
+The sections below provide details on how to set up each monitoring solution.
 
 ## Setting up monitoring using Crow
 
-Crow displays information about the projects pending analysis and the jobs currently running on Codacy.
+Crow displays information about the projects that are pending analysis and the jobs currently running on Codacy.
 
-The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
-It can be accessed through the `/monitoring` path of the url pointing to your Codacy
-installation, e.g. `http://codacy.company.org/monitoring`. You must set the
-`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
-projects can be properly established inside `crow`.
+The Crow tool is installed alongside Codacy when the Helm chart is deployed to the cluster and is configured as follows by default:
 
-For example:
+-   Crow is available on the `/monitoring` path of your Codacy installation URL, such as `http://codacy.company.org/monitoring`
+-   The default credentials to access Crow are:
 
-```yaml
-global:
-  codacy:
-    crow:
-      url: "http://codacy.example.com/monitoring"
-```
+    ```yaml
+    username: codacy
+    password: C0dacy123
+    ```
 
-### Configuring your credentials
+Follow the steps below to configure Crow and change the default configurations:
 
-We highly recommend that you define a custom password for the Crow installation. You can do this either through the `values.yaml` file or through a `--set` parameter during the Helm installation process. This parameter can be configured as follows:
+1.  Set the `global.codacy.crow.url` value in your `values.yaml` file so that Crow correctly generates anchor links to your projects. For example:
```

Comment:
To simplify the sequence of instructions, I preferred the method of editing the `.yaml` file over using the `--set` parameter on a Helm upgrade.

Reply:
Yes, using a file instead of a flag makes redeployment easier. I would suggest using a values file just for crow to make this section self-contained.

Reply:
I see, this would be an extra `values-crow.yaml` that could be appended to the other values files in a `helm upgrade` command, as we're doing for other settings.

Who would be the best person to implement this change?

Reply:
Splitting the Crow configurations into a separate .yaml file was also suggested yesterday by Caxaria and Hélio while doing a test (https://codacy.atlassian.net/browse/CY-1170).

Do you think you can do this split and I'll update the documentation accordingly, @h314to?

Reply:
I implemented this change on commit https://github.com/codacy/chart/pull/339/commits/c0892513ab7c9711d71e3952279e0ea7d9f2a67d